samples/synchronization/README.rst

```diff
@@ -44,3 +49,33 @@ Sample Output
    <repeats endlessly>
 
 Exit QEMU by pressing :kbd:`CTRL+A` :kbd:`x`.
+
+Usage for UART tracing backend
+*******************************
+Build image with "cmake -DBOARD=sam_e70_xplained -DCONF_FILE=prj_uart.conf .."
```

Comment:
Change to:
```
Build a UART-tracing image with::

   cmake -DBOARD=sam_e70_xplained -DCONF_FILE=prj_uart.conf ..
```
(Note that the two colons ``::`` at the end of the line indicate the following intented text is a code-block.)

Reply:
thanks